src/__tests__/components/frontend-engine/frontend-engine.spec.tsx

```diff
@@ -140,6 +140,30 @@ describe("frontend-engine", () => {
 		expect(onChange).toBeCalledWith(expect.objectContaining({ [FIELD_ONE_ID]: "hello" }), true);
 	});
 
+	it("should return the correct validity in the onChange prop when form is prefilled with valid values", async () => {
+		const onChange = jest.fn();
+		renderComponent(
+			{ onChange },
+			{
+				sections: {
+					section: {
+						uiType: "section",
+						children: {
+							[FIELD_ONE_ID]: {
+								uiType: "text-field",
+								label: FIELD_ONE_LABEL,
+								validation: [{ min: 5 }],
+							},
+						},
+					},
+				},
+				defaultValues: { [FIELD_ONE_ID]: "a" },
+			}
+		);
+
+		expect(onChange).toBeCalledWith(expect.objectContaining({ [FIELD_ONE_ID]: "a" }), false);
```

Comment:
the assertion doesn't match the test case description, the second argument should be `true` and the default value should have min 5 characters

Reply:
oops just fixed it, I updated the tests to cover both positive and negative outcomes